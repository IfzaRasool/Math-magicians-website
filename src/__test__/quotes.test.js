import React from 'react';
import Renderer from 'react-test-renderer';
import Quotes from '../components/quotes';
import '@testing-library/jest-dom';

it('render correctly have no items', () => {
  const quote = Renderer.create(<Quotes />).toJSON();

  expect(quote).toMatchSnapshot();
});
