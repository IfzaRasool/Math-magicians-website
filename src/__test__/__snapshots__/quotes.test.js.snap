// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly have no items 1`] = `
<div
  className="quotes"
>
  <img
    alt="quotes"
    className="quotesImg"
    src="quotes1.jpg"
  />
  <div
    className="quotes-container"
  >
    <h2>
      Math Magicians
    </h2>
    <p>
       
      Mathematics is a game played according to certain simple rules with meaningless marks on paper. David Hilbert
    </p>
  </div>
</div>
`;
